### PR TITLE
Added Definition of an AI Engineer

### PR DESCRIPTION
**Pull request by 2div**
Hey There,

I added definition of an AI Engineer in the content

Thank you

### CODE REVIEW

File: package-lock.json
Comment (offensive-vk) [inline package-lock.json:1]: Please remove the `package-lock.json` file changes.

Comment (2div) [reply package-lock.json:1]: Hi @offensive-vk 

  I removed the changes in the package-lock.json file

Best Regards

Comment (2div) [reply package-lock.json:1]: Hi @offensive-vk 

Kindly check my PR.

Thank you

Comment (offensive-vk) [reply package-lock.json:1]: You didn't remove the changes, its still modified.

Comment (2div) [reply package-lock.json:1]: Hi,

    I did added the removed lines to the file and commit it again. 
    if what i did is not correct.
    Can you please advise me how to solved ?

Thank you

Comment (offensive-vk) [reply package-lock.json:1]: I know you're smart enough to google it.

Comment (2div) [reply package-lock.json:1]: Hi @offensive-vk 

I did git revert to my first commit where the file changes happened and updated the AI Engineer content again and push it.
Please review it and let me know if what i did is correct .

Thank you

Comment (offensive-vk) [reply package-lock.json:1]: Idk what you did but its still modified in this pr. 😂

Just close this pull request and open a new one with those two files only.

Simple and easy.

Comment (2div) [reply package-lock.json:1]: Ok

**Comment by 2div** [comment]
I will close this PR due and create new PR.

**Pull request closed without merging**